poms/bom-test/pom.xml

```diff
@@ -373,6 +373,10 @@
                         <groupId>com.google.code.findbugs</groupId>
                         <artifactId>jsr305</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>io.netty</groupId>
+                        <artifactId>netty-transport-native-epoll</artifactId>
+                    </exclusion>
```

Comment:
Why exclude `netty-transport-native-epoll`?

Reply:
It was causing convergence check errors for `-redhat` version Vs non-redhat version. Given it's just a test dependency, I figured we could just exclude it.